pkg/util/validation/validate_test.go

```diff
@@ -127,9 +127,9 @@ func TestValidateLabels(t *testing.T) {
 			# HELP cortex_discarded_samples_total The total number of samples that were discarded.
 			# TYPE cortex_discarded_samples_total counter
 			cortex_discarded_samples_total{reason="label_invalid",user="testUser"} 1
-			cortex_discarded_samples_total{reason="label_name_too_long",user="testUser"} 1
-			cortex_discarded_samples_total{reason="label_value_too_long",user="testUser"} 1
-			cortex_discarded_samples_total{reason="max_label_names_per_series",user="testUser"} 1
+			cortex_discarded_samples_total{reason="tenant_label_name_too_long",user="testUser"} 1
+			cortex_discarded_samples_total{reason="tenant_label_value_too_long",user="testUser"} 1
+			cortex_discarded_samples_total{reason="tenant_max_label_names_per_series",user="testUser"} 1
```

Comment:
These are breaking changes we don't want to introduce, but won't be an issue given I suggest to roll back these changes.